blog/en/blog/2024/07/31/monthly-report.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Monthly Report (July 01 - July 31)"
+keywords: ["Apache APISIX", "API Gateway", "Monthly Report", "Contributor"]
+description: Our monthly Apache APISIX community report generates insights into the project's monthly developments. The reports provide a pathway into the Apache APISIX community, ensuring that you stay well-informed and actively involved.
+tags: [Community]
+image: https://static.apiseven.com/uploads/2024/07/31/U3v43S1b_report-july-en.png
+---
+
+> We have recently made some additions and improvements to specific features within Apache APISIX. The main improvement is moving the default values from `config-default.yaml` to a hardcoded Lua file. For detailed information, please read the monthly report.
+<!--truncate-->
+
+## Introduction
+
+From its inception, the Apache APISIX project has embraced the ethos of open-source community collaboration, propelling it into the ranks of the most active global open-source API gateway projects. The proverbial wisdom of 'teamwork makes the dream work' rings true in our way and is made possible by the collective effort of our community.
+
+From July 1 to June 31, a total of 13 contributors made 21 commits to Apache APISIX. We sincerely appreciate your contributions to Apache APISIX.
+
+## Contributor Statistics
+
+![Apache APISIX Contributors List](https://static.apiseven.com/uploads/2024/07/31/Uk3y8OVm_july-contributors.png)
+
+![Apache APISIX New Contributors](https://static.apiseven.com/uploads/2024/07/31/x59QmPpL_new-contributors-july.png)
+
+## Recent Feature Highlight
+
+- [Move `config-default.yaml` default values to hardcoded Lua file](https://github.com/apache/apisix/pull/11312)（Contributor: [bzp2010](https://github.com/bzp2010))
+
+## Recent Blog Recommendations
+
+- [Advanced URL rewriting with Apache APISIX](https://apisix.apache.org/blog/2024/07/18/advanced-url-rewrite-apisix/)
+
+- [Dynamic watermarking with imgproxy and Apache APISIX](https://apisix.apache.org/blog/2024/07/11/watermarking-infrastructure/)
```

Comment:
It might be good to add a short summary about these.

For example, a paragraph about why we moved the default configuration and an introduction/short summary of the linked blog posts.